app/controllers/api/v1/invoices_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::InvoicesController < ApplicationController
+  before_action :set_invoice, only: [:show]
+  before_action :all_invoices, only:[:index]
+  
+  # GET /invoices
+  def index 
+    render json: @invoices 
+  end
+
+  # GET /invoices/1
+  def show
+    render json:(@invoice.attributes.merge("sold_items":@invoice.sold_items))
+  end
+
+  # POST /invoices
+  def create
+    u = User.first
```

Comment:
Please use current user here. It is working. If its not let me know and we can work on fixing it.

Reply:
done